### DISCUSSION TITLE
Upgrading to 1.30.1

### DISCUSSION THREAD

**Discussion by henryhm0** [teslamate-org/teslamate#4105] [General]
Hi guys,

I am currently running version 1.29.1. If I upgrade to 1.30.1, do I need to upgrade to PostgreSQL 16?

Should I wait for the new Grafana upgrade before I update?

Thanks in advance.

**Comment by cwanja** [comment]
> If I upgrade to 1.30.1, do I need to upgrade to PostgreSQL 16?

No. 

> Should I wait for the new Grafana upgrade before I update?

No. We have rolled back Grafana and waiting for their update before we update Grafana again.

**Reply by henryhm0** [reply]
Thank you :)

**Comment by Str1atum** [comment]
Hasn't Grafana been updated already to 11.1.4 which should fix the issue?
